baseplate/lib/config.py

```diff
@@ -245,6 +245,20 @@ def open_file(text: str) -> IO:
     return open_file
 
 
+def DateTime(text: str) -> datetime.datetime:
```

Comment:
these produce timezone naive datetimes, which is a big footgun in python's datetime stack. Instead of using these functions directly I might advise a strptime call or something similar where you can specify the resultant timezone

Reply:
It would be nice if the user was required to specify the tz or the API itself  specified the tz to avoid using tz naive dates or times.